<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PageManager</class>
 <widget class="QWidget" name="PageManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="MessagePage" name="pageMsg" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="HomePage" name="pageHome" native="true">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>0</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="MinePage" name="pageMine" native="true">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>0</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
  </widget>
  <widget class="ToolBar" name="toolBar" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>520</y>
     <width>400</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
border:1px solid rgb(255, 255, 255);
border-top-color: rgb(203, 203, 203);</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ToolBar</class>
   <extends>QWidget</extends>
   <header>widgets/ToolBar.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>HomePage</class>
   <extends>QWidget</extends>
   <header>pages/HomePage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MessagePage</class>
   <extends>QWidget</extends>
   <header>pages/MessagePage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MinePage</class>
   <extends>QWidget</extends>
   <header>pages/MinePage.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
